### PR TITLE
Update API Response Panel Impression

### LINKED ISSUE #6: 03. Response Panel
> The response panel in the Firecamp API app is to view, format and get more information about the response of your request.

Response panel is not something for formatting the information, Yes it's true we can prettify but this terminology is not proper for developer language. We can simply add one line like `The response payload can be folded, prettify and copies easily from utility icons shown in below image`

> The response body can be presented as JSON or text by clicking the toggle buttons below the Body tab. With the options to copy, fold as well as word wrap if needed, the API response body can be viewed, and formatted as needed.

This is not limited to only JSON/Text, Response body can be anything like HTML/Preview, JSON/XML/Text, Image/GIF anything. so we can change this paragraph accordingly.

> Response Headers provide information about the body and can be represented as the metadata for the response.

Can give small clarity in this sentence. `Response Headers provide information about the server's response which can also be represented as the metadata for the response.`

cc @vipulgupta2048 @mehtavishwa30

### PR DESCRIPTION
**Pull request by vipulgupta2048**
Fixes #6 

Signed-off-by: Vipul Gupta (@vipulgupta2048) <vipulgupta2048@gmail.com>

### CODE REVIEW

**Comment by Nishchit14** [comment]
🚀 Deployed on https://5e75fde7be1baf3a007b30fd--firedoc.netlify.com

File: source/apis/impressions/response-panel.mdx [hunk 1]
```diff
@@ -6,19 +6,19 @@ description: Through this impression we understand more about the Response Panel
 
 ## Overview
 
-The response panel in the Firecamp API app is to view, format and get more information about the response of your request.
+The response panel in the Firecamp API app is to view the information/data sent as part of the response by the server when a request has been made.
```
Comment (Nishchit14) [inline source/apis/impressions/response-panel.mdx]: >view the information/data sent as part of 

It should be `view the information/data received as part of`

File: source/apis/impressions/response-panel.mdx [hunk 2]
```diff
@@ -6,19 +6,19 @@ description: Through this impression we understand more about the Response Panel
...
 
-The panel provides a clear eagle-eye view of your response body as well as features to dive into headers, cookies, timeline and performance benchmarks associated with your request.
+The panel provides a clear eagle-eye view of your response body as well as features to dive deeper into headers, cookies, timeline and performance benchmarks associated with your request.
 
 > Placeholder for the entire response body.
 
-The response body can be presented as JSON or text by clicking the toggle buttons below the `Body` tab. With the options to copy, fold as well as word wrap if needed, the API response body can be viewed, and formatted as needed.
+To view the the response in JSON or text, one can click the toggle buttons below the `Body` tab. Options like copy, fold and word wrap are also available through the utility icons. That way the API response body can be viewed, and styled as needed.
```
Comment (Nishchit14) [inline source/apis/impressions/response-panel.mdx]: As I mentioned in #6  It's not specific to JSON/Text. We need to write it in a generic sense.  You can try out a couple of different requests by yourself in Firecamp so you'll feel It's a complete sense of response pane. :)

Try out
1. https://google.com
2. https://api.beta.shipwire.com/exec/TrackingServices.php
3. https://jsonplaceholder.typicode.com/posts/10

image

Comment (vipulgupta2048) [reply source/apis/impressions/response-panel.mdx]: How about now

**Comment by Nishchit14** [comment]
🚀 Deployed on https://5e7d4191aa6a0c97fc669375--firedoc.netlify.com

**Pull request merged**